politeiawww/www.go

```diff
@@ -331,6 +331,14 @@ func _main() error {
 		params:          activeNetParams.Params,
 	}
 
+	// Get initial value for best block. This will subsequently be updated by
+	// the websocket connection to dcrdata.
+	bb, err := p.getBestBlockDecredPlugin()
```

Comment:
I don't like this because if dcrdata is down then you won't be able to start up politeiawww.  It will error out here and exit.  The best block should be lazy loaded in `getBestBlock()`.